src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -150,6 +150,16 @@ public function getServerParameter($key, $default = '')
         return isset($this->server[$key]) ? $this->server[$key] : $default;
     }
 
+    public function switchToAjax()
```

Comment:
Wouldn't it make more sense to use the term `XmlHttpRequest` instead of `Ajax`?

Reply:
I guess it would make more sense